share/pvss/pvss_test.go

```diff
@@ -0,0 +1,310 @@
+package pvss
+
+import (
+	"testing"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/edwards"
+	"github.com/dedis/crypto/random"
+	"github.com/dedis/crypto/share"
+)
+
+func TestPVSS(test *testing.T) {
```

Comment:
Ok I agree that this example is nicely documented and should suffice by itself. So one last request from me would be to factor out that test into an example like `pvss_example.go` which contains that and to say in the `pvss.go` top documentation to look at the example for more informations. does it make sense ? 

I'm finished after that :D